source/postprocess/depth_average.cc

```diff
@@ -294,7 +294,9 @@ namespace aspect
                              Patterns::Selection(DataOutBase::get_output_format_names().append("|txt")),
                              "The format in which the output shall be produced. The "
                              "format in which the output is generated also determines "
-                             "the extension of the file into which data is written.");
+                             "the extension of the file into which data is written. See "
+                             "the corresponding appendix of the manual for possible "
```

Comment:
I'm not sure it's clear here (and in the other places) what the corresponding appendix actually is. Maybe say "The list of possible output formats that can be given here is documented in the appendix of the manual where the current parameter is described."